Stratis.Bitcoin.Features.Consensus/PosConsensusValidator.cs

```diff
@@ -476,21 +440,23 @@ public void CheckAndComputeStake(ContextInformation context)
             // Record proof hash value.
             blockStake.HashProof = context.Stake.HashProofOfStake;
 
-            if (pindex.Height > lastCheckpointHeight)
+            if (chainedBlock.Height > lastCheckpointHeight)
```

Comment:
Did we not decide to calculate stake modifier in any case as the calculation is not a heavy operation.

Reply:
We kind of did that, although I forgot the reason why. But it is not relevant to this PR. This PR was about taking away V1 and V2 code and checkpoints are different issue. So if we still want to do that, which is possible, but I forget the reason why it would be good to do (still consumes some CPU, although not much), then we can do it, just not as a part of this PR.